sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/types/DataTypeUtils.scala

```diff
@@ -64,6 +64,8 @@ object DataTypeUtils {
    * - Both types are structs and have the same number of fields. The type and nullability of each
    *   field from read/write is compatible. If byName is true, the name of each field from
    *   read/write needs to be the same.
+   * - it is user defined type and its underlying sql type is same as the read type.
```

Comment:
- We need to capitalize the first char of the sentence.
- Do you mean `If it is` instead of `it is`?
- Also, please remove `-` if this is the continuation of the above paragraph staring line 64.

Reply:
Fixed.